src/libs/state.ts

```diff
@@ -15,11 +18,16 @@ export const authStore = Utils.atom<any>({
   fenceStatus: {},
   cookiesAccepted: undefined,
   authContext: undefined,
-  oidcConfig: {},
+  oidcConfig: {
+    authorityEndpoint: undefined,
+    clientId: undefined,
+  },
   isAzurePreviewUser: undefined,
+  sessionId: undefined,
+  sessionStartTime: {} as number,
```

Comment:
```suggestion
  sessionStartTime: -1,
```

Can this be initialized to a number instead of casting? Especially when the cast is definitely invalid: `{}` is never a number.